client/src/components/Authorization.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import axios from 'axios';
+
+
+const Authorization = {
+  isAuthenticated: false,
+  authenticate(cb) {
+    fetch('http://localhost:55555/user/login', {
+      method: 'POST',
+      body:JSON.stringify({'username':localStorage.getItem('username') , 'password':localStorage.getItem('password')})
+    })
+    .then(function(response) {
+      if(false) {
```

Comment:
here again, fix the endpoint to respond correctly and you wont need to do this things here